modules/rosa-aws-scp.adoc

```diff
@@ -1,16 +1,42 @@
 // Module included in the following assemblies:
 //
-// * rosa_getting_started/rosa-aws-prereqs.adoc
+// * rosa_architecture/rosa-sts-about-iam-resources.adoc
+//
+// * rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-aws-prereqs.adoc
+//
 
 [id="rosa-minimum-scp_{context}"]
-== Minimum required service control policy (SCP)
+= Minimum required service control policy (SCP)
+
+Service control policies (SCP) are a type of organization policy that manages permissions within your organization. SCPs ensure that accounts within your organization stay within your defined access control guidelines. These polices are maintained in the AWS Organizations and control what services are available within the attached AWS accounts. SCP management is the responsibility of the customer.
+
+Verify that your service control policy (SCP) does not restrict any of these required permissions.
+
+ifeval::["{context}" == "rosa-sts-about-iam-resources"]
+:aws-sts:
+endif::[]
+
+ifeval::["{context}" == "prerequisites"]
+:aws-non-sts:
+endif::[]
+
+ifdef::aws-sts[]
+[NOTE]
+====
+When using STS (security token service), you must ensure that the service control policy does not block the following resources:
```

Comment:
A quick Atom search shows _security token service (STS)_ always. And most are prefixed with _AWS_.